libmscore/score.cpp

```diff
@@ -2295,11 +2295,19 @@ bool Score::appendScore(Score* score)
             return false;
       int tickOfAppend = lastMeasure->endTick();
 
-      if (!lastMeasure->lineBreak() && !lastMeasure->pageBreak()) {
-            lastMeasure->undoSetBreak(true, LayoutBreak::Type::LINE);
+      if (addPageBreak) {
+            // add PageBreak if one doesn't already exist
+            if (!lastMeasure->pageBreak())
```

Comment:
OTH if the user specified 'add page break' in the dialog (s)he should get it, shouldn't (s)he?
But you can't have a line break and a page break at the same place, at least not via the palette, so if anything, the page break should **replace ** the line break.